ext/uri/config.m4

```diff
@@ -0,0 +1,27 @@
+dnl Configure options
+dnl
+PHP_ARG_ENABLE([uri],
+  [for URI support],
+  [AS_HELP_STRING([--enable-uri],
+    [Enable URI support])])
+
+if test "$PHP_URI" != "no"; then
+    PHP_LEXBOR_CFLAGS="-I$abs_srcdir/ext/dom/lexbor -DLEXBOR_STATIC -I$abs_srcdir/ext/uri/uriparser/include"
```

Comment:
This effectively makes ext/dom a dependency of ext/uri. Should Lexbor be moved somewhere else?

Reply:
It's just a WIP behavior from the early days of the PR. Once I manage to make the RFC approved, I'll create a lexbor "extension" (similarly to mysqlnd).